tests/e2e/translations/nepali-dates-and-numbers.wdio-spec.js

```diff
@@ -37,13 +38,50 @@ const setExistentReportDates = async (dates) => {
   return utils.saveDocs(reports);
 };
 
+const momentToBikYMD = (mDate) => {
+  const bsDate = bikramSambat.toBik(moment(mDate).format('YYYY-MM-DD'));
+  return `${bsDate.year}-${bsDate.month}-${bsDate.day}`;
+};
+
+const formId = 'B';
+const formTitle = 'LMP with BS Date';
```

Comment:
It looks like this is not reused somewhere, you can just inline it in the form definition.

Reply:
Done. Thanks.